src/orders/delivery/app.py

```diff
@@ -0,0 +1,110 @@
+# Python's libraries
+from typing import Dict
+from typing import Any
+import re
+
+# import json
+
+# Own's modules
+from location_router import TravelPlanner
+from delivery_modules.dao.order_dao import OrderDAO
+from delivery_modules.processors.order_helpers import OrderProcessor
+from delivery_modules.utils.doorman import DoormanUtil
+from delivery_modules.errors.auth_error import AuthError
+
+from settings import ORDERS_PRIMARY_KEY
+
+# Third-party libraries
+from aws_lambda_powertools import Logger
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+
+def set_delivery_schedule_order(
+    event: Dict[str, Any], context: LambdaContext
+) -> Dict[str, Any]:
+    """This function will retrieve todays orders to generate an optimized path.
+
+    :param event: Custom object that can come from an APIGateway, nothing relevant in the body.
```

Comment:
I would change it to : 
Custom object that can come from an APIGateway, containing date field for delivery orders.